release notes/v0.9.0.md

```diff
@@ -0,0 +1,111 @@
+k6-browser `v0.9.0` is here 🎉! This release includes:
+
+- A few breaking changes.
+- `browserContext.addCookies` has now been implemented.
+- `browserType.Connect` has now been implemented and so k6-browser can connect to an already running Chrome/Chromium browser instance.
+- Web vitals are natively supported by the k6-module.
```

Comment:
What are we referring here with `k6-module`?

Reply:
Yep, correct, resolved in ca916eac3918873fcdace84c06d9d173c0e5ba0c